websites/demo-react-image-next/src/app/page.tsx

```diff
@@ -1,28 +1,64 @@
 "use client";
 
-import { Load } from "@fepack/react-image";
+import { Load, extractRGBAs } from "@fepack/react-image";
 import { ErrorBoundary, Suspense } from "@suspensive/react";
 import Link from "next/link";
+import { useState } from "react";
 import { Spinner } from "~/components/uis";
 
 export default function Home() {
+  const [size, setSize] = useState(8);
+
   return (
     <div>
       <Link href="/load">to load</Link>
       <ErrorBoundary fallback={() => <>load image error</>}>
         <Suspense.CSROnly fallback={<Spinner />}>
           <Load src="https://ik.imagekit.io/ikmedia/women-dress-2.jpg">
-            {(loaded) => <img src={loaded.src} />}
+            {(loadedImage) => <img src={loadedImage.src} />}
```

Comment:
what about `image`? Since it is `Load`, I naturally think it is a loaded image.

```suggestion
            {(image) => <img src={image.src} />}
```